DoSomething/quasar
# Combine northstar backfill

#### What's this PR do?
refactor: reuse shared code for full and partial backfill of northstar users

#### Where should the reviewer start?
Understand `process_all_pages` method in DSNorthstarScraper.py. read the rest of code changes.

#### How should this be manually tested?
Hacky: In DSNorthstarScraper.py, comment out line 66. Add a new line at 66: `print(params)`. Run the jobs and see the params print. You should be able to see the page increment. This is the request that is made to the API.

For partial backfill, try setting default `limit` to 20 (on line 55). Run the job and you should see the hourly intervals, and page increment.

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
